parser/parser.go

```diff
@@ -10,12 +10,12 @@ func SnippetToRawAST(snippet, fileName string) (ast.Node, ast.Fodder, error) {
 	return parser.SnippetToRawAST(ast.DiagnosticFileName(fileName), fileName, snippet)
 }
 
-func SnippetToAst(snippet, fileName string) (ast.Node, error) {
-	return program.SnippetToAST(ast.DiagnosticFileName(fileName), fileName, snippet)
+func SnippetToAst(snippet, fileName string, globalVars ...ast.Identifier) (ast.Node, error) {
+	return program.SnippetToAST(ast.DiagnosticFileName(fileName), fileName, snippet, globalVars...)
 }
```

Comment:
Staticka analyza je samostatna funkcia, nie je definovana nad `VM` a preto musime dodat globalne premenne cez parametre, je to pridane na viacerych miestach.